### PR TITLE
add zh_TW support

### PR DESCRIPTION
**Pull request by johnrosen1**

### CODE REVIEW

File: localization/zh_TW.inc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	'cd_use_categories' => "創建新的聯繫人組時，使用類別而不是KIND = group（RFC6350）\n",
+	'cd_delete' => "刪除此通訊錄",
+	'cd_name_new' => "配置新地址簿",
+	'cd_refresh_time' => "更新間隔（小時）",
+	'cd_err_noabfound' => '找不到通訊錄',
+	'cd_enabled' => '已啟用',
+	'cd_disabled' => '已禁用',
+	'cd_lastupdate_time'=>'最後更新時間',
+	'cd_resync' => "強制與服務器立即同步",
+	'cd_msg_synchronized' => 'CardDAV地址簿$ name同步（花費 $duration秒）',
```
Comment (mstilkerich) [inline localization/zh_TW.inc:19]: There is a space in `$ name` that will prevent proper expansion of the variable. Please remove the space.

**Review by mstilkerich — CHANGES_REQUESTED** [review]
Thanks for your contribution! There is a tiny issue in one of the messages, please fix.

File: localization/zh_TW.inc (hunk 1 above)
Comment (johnrosen1) [reply localization/zh_TW.inc]: done

**Review by mstilkerich — COMMENTED** [review]
Thanks!

**Pull request merged**